packages/frenetic/frenetic.5.0.5/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "The Frenetic Programming Language and Runtime System"
+maintainer: "Nate Foster <jnfoster@cs.cornell.edu>"
+authors: "Arjun Guha <arjun@cs.umass.edu>, Nate Foster <jnfoster@cs.cornell.edu>, Steffen Smolka <smolka@cs.cornell.edu>"
+homepage: "http://frenetic-lang.org"
+license: "LGPL-3.0-or-later"
+bug-reports: "https://github.com/frenetic-lang/frenetic/issues"
+dev-repo: "git+https://github.com/frenetic-lang/frenetic.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "ocaml" {build & >= "4.11.0"}
+  "dune" {build & >= "2.0"}
```

Comment:
```suggestion
  "dune" {>= "2.0"}
```
Dune does not support the `build` tag. See https://github.com/ocaml/dune/issues/2147

Reply:
Fixed, thanks.